mulhome_scripts/delete_all_decoupled_pricing_circe.py

```diff
@@ -75,44 +78,47 @@ def delete_all_decoupled_pricing_circe(app_name):
     try:
         resp = extensions_v1_beta1_api.read_namespaced_deployment(home_name, namespace)
     except ApiException as e:
-        print("No Such Deplyment Exists")
+        logging.debug("No Such Deplyment Exists")
 
     # if home exists, delete it 
     if resp:
         del_resp_0 = extensions_v1_beta1_api.delete_namespaced_deployment(home_name, namespace=namespace, body=v1_delete_options)
-        print("Deployment '%s' Deleted. status='%s'" % (home_name, str(del_resp_0.status)))
+        logging.debug("Deployment '%s' Deleted. status='%s'" % (home_name, str(del_resp_0.status)))
 
     # Check if there is a replicaset running by using the label app=home
     # The label of kubernets are used to identify replicaset associate to each task
     label = "app="+app_name+"-home"
     resp = extensions_v1_beta1_api.list_namespaced_replica_set(label_selector = label,namespace=namespace)
     # if a replicaset exist, delete it
     
-    # print resp.items[0].metadata.namespace
+    # logging.debug resp.items[0].metadata.namespace
     for i in resp.items:
         if i.metadata.namespace == namespace:
-            del_resp_1 = extensions_v1_beta1_api.delete_namespaced_replica_set(i.metadata.name, namespace, v1_delete_options)
-            print("Relicaset '%s' Deleted. status='%s'" % ('home', str(del_resp_1.status)))
+            # del_resp_1 = extensions_v1_beta1_api.delete_namespaced_replica_set(i.metadata.name, namespace, v1_delete_options)
```

Comment:
Delete commented code. It looks like these Mulhome scripts are using similar boilerplate code. This could be wrapped into a library instead. Since these are control scripts, maybe we can set this as a low priority because I don't see these scripts changing so often.

Reply:
I think it depends on the kubernetes client installed version so they call different versions (?!?). I tested on both laptops and I have to use 2 different versions. Have to check this again.